databricks_cli/sdk/service.py

```diff
@@ -616,15 +616,22 @@ class SecretService(object):
     def __init__(self, client):
         self.client = client
 
-    def create_scope(self, scope, initial_manage_principal=None, scope_backend_type=None,
+    def create_scope(self, scope, initial_manage_principal=None, scope_backend_type=None, azure_keyvault=None,
                      headers=None):
         _data = {}
         if scope is not None:
             _data['scope'] = scope
         if initial_manage_principal is not None:
             _data['initial_manage_principal'] = initial_manage_principal
         if scope_backend_type is not None:
-            _data['scope_backend_type'] = scope_backend_type
+            if scope_backend_type == 'databricks':
+                _data['scope_backend_type_str'] = 1
+            elif azure_keyvault is not None and scope_backend_type == 'azure_keyvault':
+                _data['scope_backend_type'] = 2
```

Comment:
Just making sure, `scope_backend_type` and `scope_backend_type_str` differ by design right?

Reply:
Can we put in the strings `DATABRICKS` and `AZURE_KEYVAULT` in the _data[`scope_backend_type`] field directly? Our rpc system should be able to handle the string constants as part of the enum I think.

Also we should not make these changes in this file since it is autogenerated. Could you lift them to `databricks_cli/secrets/api.py` and regenerate this file to include the `backend_azure_keyvault` argument?